backend/service/github/github.go

```diff
@@ -216,6 +218,10 @@ func boolPtr(b bool) *bool {
 	return &b
 }
 
+func intPtr(i int) *int {
+	return &i
+}
```

Comment:
It looks like this is only used by the tests. Can you move it into that file instead?

Reply:
done!